new-lamassu-admin/src/pages/Cashout/Wizard.js

```diff
@@ -54,12 +54,22 @@ const Wizard = ({ machine, locale, onClose, save, error }) => {
     {
       type: 'top',
       display: 'Cassette 1 (Top)',
-      component: Autocomplete
+      component: Autocomplete,
+      inputProps: {
+        options: R.map(it => ({ code: it, display: it }))(options),
+        labelProp: 'display',
+        valueProp: 'code'
+      }
```

Comment:
will this work even if we don't have options? There's a scenario where this should be a simple InputNumber, not sure where that's implemented.